src/IssueViz/Editor/SelectedIssueTagging/Adornment/IssueLocationAdornmentTaggerProvider.cs

```diff
@@ -66,7 +66,7 @@ internal class IssueLocationAdornmentTaggerProvider : IViewTaggerProvider
                 return null;
             }
 
-            var tagger = textView.Properties.GetOrCreateSingletonProperty(typeof(IssueLocationAdornmentTagger), () => Create(textView as IWpfTextView));
+            var tagger = Create(textView as IWpfTextView);
```

Comment:
Same change for all of the non-singleton taggers - always return a new tagger instance when asked.